test/hermetic_taskrun_test.go

```diff
@@ -91,3 +110,35 @@ apt-get install -y curl`,
 		},
 	}
 }
+
+func unpriviligedTaskRun(name, namespace, executionMode string) *v1beta1.TaskRun {
+	return &v1beta1.TaskRun{
+		ObjectMeta: metav1.ObjectMeta{Name: name,
+			Namespace: namespace,
+			Annotations: map[string]string{
+				"experimental.tekton.dev/execution-mode": executionMode,
+			},
+		},
+		Spec: v1beta1.TaskRunSpec{
+			Timeout: &metav1.Duration{Duration: time.Minute},
+			TaskSpec: &v1beta1.TaskSpec{
+				Steps: []v1beta1.Step{
+					{
+						Container: corev1.Container{
+							Name:  "curl",
+							Image: "gcr.io/cloud-builders/curl",
+							SecurityContext: &corev1.SecurityContext{
```

Comment:
thanks @priyawadhwa for the changes, can this `SecurityContext` be configured for a pod?

Reply:
we can include an additional test with security context at the pod level ...